bin/phase-helpers.sh

```diff
@@ -1014,6 +1014,7 @@ if ___eapi_has_eapply; then
 		local i found_doublehyphen
 		# first, try to split on --
 		for (( i = 1; i <= ${#@}; ++i )); do
+			# shellcheck disable=SC2199
```

Comment:
Why not use `${@[i]}` instead?

Reply:
Are referring to the line below, containing `[[ ${@:i:1} == -- ]]`? It seems like `shellcheck` is is not parsing the script properly.

Reply:
Actually `${*:i:1}` will work here, as shellcheck suggests (and `${@[i]}` is invalid syntax).

I suspect that the shellcheck authors would agree that `${@:i:1}` is valid in this context, since the length of the slice is exactly 1, so there's no implicit concatenation happening.